.github/workflows/ci.yml

```diff
@@ -158,6 +158,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install wheel pytest pytest-cov nvidia-pyindex
+          python -m pip install git+https://github.com/HazyResearch/flash-attention.git
```

Comment:
please also add this package in `setup.py`

Reply:
That's a problem. `setup.py` cannot install packages via git. `flash attn` has not publish pip yet.

Reply:
Then, we should also check whether `flash-attention` is installed when we want to use it, like:
```
try:
    import flash_attn
except:
    FLASH_ATTENTION_AVAILABLE = False
```